apps/judicial-system/backend/src/app/modules/case/models/dateLog.model.ts

```diff
@@ -19,6 +19,16 @@ import { Case } from './case.model'
   timestamps: false,
 })
 export class DateLog extends Model {
+  static arraignmentDate(dateLogs?: DateLog[]) {
```

Comment:
This is a new pattern, right? I like it :D

Reply:
Yes it is new. I may move this to the lib later as the same logic is being used in the api layer. For now, I will keep it here as dates are treated as strings in the api.